docs/quickstart.md

```diff
@@ -0,0 +1,765 @@
+=== "app.py"
+
+    ```python hl_lines="1 11 13 19 21 27"
+    from aws_lambda_powertools import Logger, Tracer
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+    from aws_lambda_powertools.logging import correlation_paths
+
+    logger = Logger(service="APP")
+    tracer = Tracer()
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    @tracer.capture_method
+    def hello_name(name):
+        tracer.put_annotation("User", name)
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    @tracer.capture_method
+    def hello():
+        tracer.put_annotation("User", "unknown")
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+    @tracer.capture_lambda_handler
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+    
+Thanks to the help of powertools tracer we have much cleaner code right now. 
+To make our methods visible in the traces, we added `@tracer.capture_method` decorator to the processing methods. 
+We added annotations directly in the code without adding it with the context handler using the `tracer.put_annotation` method. 
+Since we added the `@tracer.capture_lambda_handler` decorator for our `lambda_handler`, powertools automatically adds cold start information as an annotation. 
+It also automatically adds lambda response as a metadata into trace, so we don't need to worry about it as well.
+!!! tip 
+    For differences between annotations and metadata in traces, please follow [link](https://awslabs.github.io/aws-lambda-powertools-python/latest/core/tracer/#annotations-metadata). 
+Therefore, you should see traces of your lambda in the X-ray console.
+=== "Example X-RAY Console View"
+![Tracer utility](./media/tracer_utility_showcase_2.png)
+
+You may also consider using **CloudWatch ServiceLens** which links the CloudWatch metrics and logs, in addition to traces from the AWS X-Ray.
+It gives you a complete view of your apps and their dependencies, making your services more observable. 
+From here, you can easily browse to specific logs in CloudWatch Logs Insight, Metrics Dashboard or Traces in CloudWatch X-Ray traces.
+=== "Example CloudWatch ServiceLens View"
+![CloudWatch ServiceLens View](./media/tracer_utility_showcase_3.png)
+!!! Info
+    For more information on CloudWatch ServiceLens, please visit [link](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ServiceLens.html).
+## Custom Metrics
+The final step to provide complete observability is to add certain measures at the business level. 
```

Comment:
certain measures -> metrics (or business metrics). You can also drop one or two examples such as number of sales or reservations.

Reply:
Added `The final step to provide complete observability is to add business metrics (such as number of sales or reservations).`